### PR TITLE
Update default lease get all to include owned leases

### PR DESCRIPTION
**Pull request by tzumainn**
Previously, getting all leases defaulted to only showing the user
leases that they were lessees for. This change adds leases that
the user is the owner for.

### CODE REVIEW

File: esi_leap/api/controllers/v1/lease.py [hunk 1]
```diff
@@ -155,26 +155,27 @@ def _lease_get_all_authorize_filters(cdict,
...
                 possible_filters['project_id'] = project_id
             else:
                 if project_id is None:
                     project_id = cdict['project_id']
-                elif project_id != cdict['project_id']:
-                    policy.authorize('esi_leap:lease:lease_admin',
-                                     cdict, cdict)
-
-                possible_filters['project_id'] = project_id
+                    possible_filters['project_or_owner_id'] = project_id
```
Comment (ljmcgann) [inline esi_leap/api/controllers/v1/lease.py:173]: is github being weird or is this tabbed in when it shouldnt be

**Review by ljmcgann — APPROVED** [review]
looks good to me

File: esi_leap/api/controllers/v1/lease.py (hunk 1 above)
Comment (tzumainn) [reply esi_leap/api/controllers/v1/lease.py:173]: Ah, the indent is correct - we only set 'project_or_owner_id' if the user has decided *not* to filter by project_id. Thanks for the review!

Comment (ljmcgann) [reply esi_leap/api/controllers/v1/lease.py:173]: oh i was being dumb and read the old line with the new else statement and was confused. All else considered yes it looks fine.

**Review by ljmcgann — APPROVED** [review]

**Pull request merged**